lib/yardstick/v2_client/grant_status.rb

```diff
@@ -0,0 +1,24 @@
+module Yardstick
+  module V2Client
+    class GrantStatus
+      include RemoteModel
+
+      attr_accessor :response # TODO: refactor this up so it's available to other models
+
+      resource_uri '/v2/grant_status'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.